data-source/jsonnet/northern-ireland/blocks/individual/employment/work_location.jsonnet

```diff
@@ -39,17 +39,17 @@ local question(title) = {
   ],
 };
 
-local nonProxyTitleWork = 'What address do you travel to for your main place of work?';
+local nonProxyTitleWork = 'What is the address of your main place of work?';
 local proxyTitleWork = {
-  text: 'What address does <em>{person_name}</em> travel to for their main place of work?',
+  text: 'What is the address of <em>{person_name}</em>\'s main place of work?',
```

Comment:
We shouldn't be doing possessiveness like this, Welsh doesn't have `'s`. We already have one in `rules.libsonnet` for handling this. `personNamePossessive`

Same applies to all other cases in this PR.